### ISSUE TITLE
biosample_category_enum updates

### ISSUE THREAD

**Issue by turbomam** [microbiomedata/nmdc-schema#481]
- name too general
- add SIP annotations
- change meanings to see_alsos

**Comment by turbomam** [comment]
@mslarae13 please elaborate

**Comment by mslarae13** [comment]
@turbomam  is this still relevant? Has it been done or been captured elsewhere?

**Comment by aclum** [comment]
As discussed at the metadata meeting to address that 'name too general' we decided to do the following:
- [ ] depreciate biosample_categories slot on class Biosample in favor of funding_category and site_category. 
- [ ] Create new enumerations FundingCategoryEnum and SiteCategoryEnum
- [ ] FundingCategoryEnum permissible values will be SFA and FICUS
- [ ] SiteCategoryEnum permissible values will be NEON and LTER
- [ ] Remove SIP
- [ ] Move values currently in meaning to see_also
- [ ] data in mongo (biosample_set) will need to be migrated since biosample_categories has been used

**Comment by aclum** [comment]
This will require data migration, biosample ingest pipeline code update and data portal ingest code update so this needs to be carefully coordinated.

**Comment by turbomam** [comment]
slots `funding_category` and `site_category` could be assigned to some `slot_group` or `subset` to emphasize that they can be used for some common search-ability goal

- https://linkml.io/linkml-model/latest/docs/slot_group/
- https://linkml.io/linkml-model/latest/docs/is_grouping_slot/
- https://linkml.io/linkml-model/latest/docs/in_subset/
- https://linkml.io/linkml-model/latest/docs/SubsetDefinition/

**Comment by ssarrafan** [comment]
@aclum @mslarae13 @turbomam is this issue still a priority?  Can it be closed? Are there other issues already?

**Comment by aclum** [comment]
Ill defer to @mslarae13  we may want to wait for the renewal modeling of sites.

**Comment by mslarae13** [comment]
I agree with @aclum considering the decision made 2 years ago to add other grouping slots might be different with renewal work. 

I suspect we will eventaully close this issue as not planned an defer to more granular changes for the renewal. 
Not a priority atm
